pkgs/top-level/python-packages.nix

```diff
@@ -8470,7 +8470,7 @@ in {
 
   simplekml = callPackage ../development/python-modules/simplekml { };
 
-  simple_di = callPackage ../development/python-modules/simple_di { };
+  simple-di = callPackage ../development/python-modules/simple_di { };
```

Comment:
Please rename the directory, too.

Reply:
There seem to be quite a few directories that have underscores in them; should those be normalized as well?